index.js

```diff
@@ -106,6 +113,12 @@ module.exports = postcss.plugin('postcss-css-variables', function(options) {
 				});
 				variableRootRule.append(varDecl);
 
+				// mark JS-injected variables for removal if `preserveInjectedVariables`
+				// option is passed.
```

Comment:
// Colect JS-injected variables for removal if `opts.preserveInjectedVariables = false`